api-docs/cloud-load-balancers-v1/api-reference/methods/delete-bulk-delete-load-balancers-v1.0-account-loadbalancers.rst

```diff
@@ -14,6 +14,10 @@ The bulk-delete load balancer function removes the specified load balancer and
 its associated configuration from the account. Any and all configuration data is
 immediately purged and is not recoverable.
 
+To bulk-delete load balancers, provide a query parameter list of load balancer IDs
```

Comment:
To bulk-delete load balancers, provide a query parameter list of load balancer IDs.